steps/run-android-instrumentation-tests/1.0.0/step.yml

```diff
@@ -0,0 +1,136 @@
+title: Run instrumentation tests
```

Comment:
This name (and step id) suggests rather running instrumentation tests on devices/emulators connected to the bitrise VM via adb than using some external service.
The name should be "Run instrumentation tests on emulator.wtf" or something similar.

Reply:
Also please update the id of the step in step library from `run-android-instrumentation-tests` to something like `emulator-wtf-android-test`